pages/docs/security/securing-fossbilling.mdx

```diff
@@ -53,6 +53,20 @@ These following documents security options under the `config.php` configuration
 
 ## Cloudflare
  - Enable `IP Geolocation` under your website's `Network` settings. This will allow FOSSBilling to use a visitor's country (based on IP address) to help prevent session hijacking.
+-  Allow only /admin/staff/login page for country or IP using Cloudflare WAF
+
+This rule allows access to the /admin/staff/login page for a specific country and IP while blocking others.
+
+Expression Builder for Country Block:
+`(http.request.uri.path contains "/admin/staff/login" and ip.geoip.country ne "US")`
```

Comment:
It might be a good idea to protect the actual login API endpoint as well which would be `/api/guest/staff/login`.
Otherwise the API endpoint can be hit to authenticate without ever reaching the login page